src/Mocking/JSInterop/MissingMockJsRuntimeException.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Egil.RazorComponents.Testing
+{
+    /// <summary>
+    /// Exception use to indicate that a MockJsRuntime is required by a test
+    /// but was not provided.
+    /// </summary>
+    [SuppressMessage("Design", "CA1032:Implement standard exception constructors", Justification = "<Pending>")]
+    public class MissingMockJsRuntimeException : Exception
+    {
+        /// <summary>
+        /// Identifer string used in the JSInvoke method.
+        /// </summary>
+        public string Identifier { get; }
+
+        /// <summary>
+        /// Arguments passed to the JSInvoke method.
+        /// </summary>
+        public object[] Args { get; }
```

Comment:
```suggestion
        public IReadOnlyList<object> Arguments { get; }
```
This should get rid of the CA1819 warning, and the user has no need to change the args array anyway. Also, I prefer to write out the full name. Args could be misinterpreted by some non-native english speakers.

Reply:
Sure. For what its worth, I used `args` because its the name used in the IJsRuntime interface.